src/mrpro/data/_KData.py

```diff
@@ -47,6 +47,18 @@
 
 @dataclasses.dataclass(slots=True, frozen=True)
 class KData:
+    """MR raw data / k-space data class.
```

Comment:
Didnt we at some point decide to do 

header: KHeader
"""MR raw data header (KHeader) containing required meta data"""

data: torch.Tensor
"""..."""

to document attributes? do we then need a docstring in the class that does the same?
I would prefer to have a docstring on the attributes instead of on the class.

Reply:
I removed the `Attributes` in the docstring.
The documentation in the style you pointed out will be in the future when we do this for all classes I would say.